polliwog/segment/test_segment_functions.py

```diff
@@ -203,9 +203,12 @@ def test_closest_point_of_line_segment():
             p2,
         ]
     )
-    closest_points = closest_point_of_line_segment(
+    expected_t_values = np.array([40 / 48, 28 / 48, 0, 0, 1, 1, 1, 1])
```

Comment:
(I did not compute these values by hand... I inspected what the t_value computation was doing, and reproduced that here)